File: PPA/DailyFlash/August/20August/Program2.c


#include<stdio.h>

void main() {

	float principle, time, interest, SI;

	printf("Enter the principle amount, time and interest rate: ");
	scanf("%f%f%f",&principle,&time,&interest);

	SI = (principle*time*interest)/100;

	printf("The simple Interest is %.2f\n",SI);
}

File: Java/DailyFlash/August/20August/Program2.java

import java.util.Scanner;

class SimpleInterest {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		float principle_amt, rate, time, SI;

		System.out.print("Enter the principle_amt, rate of interest and time: ");
		principle_amt=sc.nextFloat();
		rate=sc.nextFloat();
		time=sc.nextFloat();

		SI=(principle_amt*rate*time)/100;

		System.out.println("Simple interest is "+SI);
	}
}


